CHARTER.md

```diff
@@ -0,0 +1,127 @@
+
+### Governing Board Election
+Governing Board elections are announced at least four weeks in advance.
+
+Nomination process:
+
+ * A person can nominate themselves as a candidate for the Governing Board.
+ * Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
+
+ * Governing Board elections take place every year.
+ * Each Voting Member ranks all nominees according to ranked-choice voting.
+ * The nominee with the largest number of votes gets appointed, the nominee with the second largest number of votes gets appointed, and so on.
+ * In case there are multiple nominees per Third-party organization, only the one with the largest number of votes gets appointed.
+ * In case of a tie where more nominees than remaining seats receive an equal number of votes, the vote shall be repeated for the remaining seats.
+
+### Terms of Governing Board Members
+Governing Board members will serve for a one-year term.
+
+### Governing Board Member Remuneration
+Governing Board members receive no remuneration by the MapLibre Organization for their services on the Board, other than reimbursement for the out-of-pocket expenses incurred when carrying out activities on behalf of the Governing Board. Such expenses must be pre-authorized by at least two other board members.
+### Conflicts of Interest
+The Governing Board develops a conflict of interest policy which details how Governing Board members can disclose conflicts of interest between their role as MapLibre Governing Board members and other professional and personal activities. 
+
+The policy shall be reviewed on a regular basis and be publicly available. 
+
+Board members with a conflict of interest with regards to a give matter abstain from votes connected to it.
+### Replacement of Governing Board Members
+In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
+
+### Snap Election
+
+A snap election of the Governing Board, i.e. an election which takes place earlier than the usual one year period, can be called if 50 percent of Voting Members who participated in the last election sign a no-confidence motion.
+
+If a no-confidence motion passes, the current Governing Board shall until the next election not take any major decision, such as but not limited to:
+
+* authorize new expenditures,
+* make any changes to personnel.
+
+Within two weeks, people can register themselves for the election with a self-nomination. Two weeks after the self-nomination deadline, the snap election shall take place, i.e. four weeks after a no-confidence motion passes.
+
+## Changes to the MapLibre Charter
+
+* Voting Members can propose changes to the MapLibre Charter.
+* Changes to the Charter require a Voting Members vote with a two-thirds majority.
```

Comment:
2/3 of all eligible Voting Members or 2/3 of those who voted? IMO, it needs to be the latter, explicitly, or we paint ourselves into a corner in the future and can never amend the charter.

Reply:
Let's always use the majority (be it 50 percent or 2/3 or whatever) of the people who voted.

Reply:
Let's always use the majority (be it 50 percent or 2/3 or whatever) of the people who voted.

Reply:
And use sufficiently long notice periods such that people can vote...

Reply:
Specified the majority to be 2/3 of the people who participate in the vote.